homework/04LessonFour.js

```diff
@@ -62,7 +75,15 @@ var problemNineObject = {
 };
 // update 'fav food' so that its value is tacos
 // update city so that its value is austin
+  var problemNineObject = {
+
+    'fav food': 'tacos',
+    city: 'austin'
+
+  };
```

Comment:
You need to use your knowledge of dot and bracket notation to manipulate the problemNineObject object. Thus you wouldn't be declaring the same variable twice.

Ex:
`obj['key 1'] = "changed!"`